Sources/Datadog/RUM/RUMMonitor/RUMContext.swift

```diff
@@ -0,0 +1,21 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal struct RUMContext {
+    /// An ID of RUM application.
+    let rumApplicationID: String
+    /// An ID of current RUM session. May change over time.
+    var sessionID: UUID
+
+    /// An ID of currently displayed view.
+    var activeViewID: UUID?
+    /// An URI of currently displayed view.
+    var activeViewURI: String?
+    /// An ID of active user action.
+    var activeUserActionID: UUID?
+}
```

Comment:
the main idea is sharing the context among scopes, isn't it?
if we make this a `class`, the same context will be shared and we won't have to compute it every time as shown in RFC, wdyt?

Reply:
The idea is to propagate the context, not "share" it. Sharing (with `class`) won't work, i.e. given this RUM state:
```
RUMMonitor
| -- Application Scope
     |-- Session Scope
         |--- View A Scope
              |--- Resource A1 Scope
              |--- ...  
         |--- View B Scope
              |--- Resource B1 Scope
              |--- ...
```
The `Resource A1 Scope` and `Resource B1 Scope` can't share the same `RUMContext`, as their `activeViewID` and `activeViewURI` will be different (`viewA.id` + `viewA.uri` vs `viewB.id` + `viewB.uri`). Same for `activeUserActionID`: one view may have an active action, while the other doesn't.

Now, the question is _"when a Session Scope may have two active View scopes?"_. There example cases:
* when master - details View Controller hierarchy is presented on iPad,
* when user is instrumenting `viewDidLoad()` / `viewDidDisappear` in his view controllers (the `.startView` and `.stopView` commands will be delivered in a wrong order).